src/db/entry.rs

```diff
@@ -18,15 +18,25 @@ pub struct Entry {
     pub tags: Vec<String>,
 
     pub times: Times,
+
+    pub custom_data: CustomData,
+
+    pub icon_id: Option<usize>,
+    pub custom_icon_uuid: Option<String>,
```

Comment:
@sseemayer this could be refactored into a single enum, since those should never be used at the same time I believe.

Reply:
I thought so too, but according to the KeePass sourcecode (which I cannot link into because it is served in a zip file from SourceForge, not even from subversion 😱 , see e.g. `KeePassLib/Serialization/KdbxFile.Write.cs:511`), `<IconId>` will always be present, and `<CustomIconUuid>` will be optionally present.

Reply:
> which I cannot link into because it is served in a zip file from SourceForge, not even from subversion

:scream: indeed